src/components/DatePickerInput.vue

```diff
@@ -92,24 +94,24 @@ export default {
       default: false
     },
   },
+  data () {
+    return {
+      dateMask: 'YYYY-MM-DD',
```

Comment:
Why hard-code this?
Can’t we use some strategy to format depending on user locale? We have many tools for this: Quasar, ICU…

If there are constraints it would be good to document them.

Reply:
In `DatePickerInput` component, we expect the property `date` to be in ISO format.
It is passed then to `QDate` which I used the above mask.

The reason I hard-coded the mask with the value `YYYY-MM-DD` is because we work with dates in ISO format for the logic. This mask is used to format the date for `QDate` (default expected format is `YYYY/MM/DD`) and the selected date is returned as a string with this format.

By setting the format in ISO format, we can save up this format step as it is already in the right format.
Concerning the internationalization, we already used ICU in `DatePickerInput` component.
ICU also expect a JS Date object so another formatting is needed if the format isn't ISO.
https://github.com/stelace/marketplace-demo/blob/3715a9caa99bf326d28dd0b514600916ee059be5/src/components/DatePickerInput.vue#L103-L106

Reply:
Thanks, could be useful for future reader.